web/package.json

```diff
@@ -15,7 +15,9 @@
     "lint": "ng lint",
     "reinstall": "rm -rf node_modules tmp deploy dist && npm install && npm rebuild node-sass",
     "reinstall:latest": "rm -rf node_modules tmp deploy dist && npm install",
-    "test": "npm run lint && npm run build:dev && concurrently -k -s first \"npm start\" \"sleep 15 && npm run e2e:ci\""
+    "test": "npm run lint && npm run build:dev && concurrently -k -s first \"npm start\" \"sleep 15 && npm run e2e:ci\"",
```

Comment:
I dont think `npm run e2e:ci` exists?

Reply:
I have not added this command

Reply:
oh the line was already there 🤦‍♂ , the comma made it look like the whole line had changed.

I think something like this should work...
```
"start:test": "serve -s -p 8080 dist",
"test": "npm run lint && npm run build && concurrently -r -k -s first \"npm run start:test\" \"npm run cypress:run\"",
```